### PR TITLE
added listAccountSas TLE

### PR DESCRIPTION
**Pull request by bmoore-msft**

### CODE REVIEW

File: assets/ExpressionMetadata.json [hunk 1]
```diff
@@ -247,9 +247,21 @@
...
         },
+        {
+            "name": "listAccountSas",
+            "expectedUsage": "listAccountSas(resourceId, apiVersion, requestContent)",
+            "description": "Creates and returns a SAS token with the specified permissions and expiry from the requestContent. The requestContent object must contain the signedServices, signedResourceTypes, signedPermission and signedExpiry properties.",
+            "minimumArguments": 3,
+            "maximumArguments": 3,
+            "returnValueMembers": [
+                {
+                    "name": "accountSasToken"
```
[the remark this answers is not in the document]
Comment (bmoore-msft) [reply assets/ExpressionMetadata.json]: It returns an object with exactly one property named accountSasToken

File: assets/ExpressionMetadata.json [hunk 2]
```diff
@@ -247,9 +247,21 @@
...
+                }
+            ]
+        },
         {
             "name": "listKeys",
-            "expectedUsage": "listKeys(resourceName\/resourceIdentifier, apiVersion)",
+            "expectedUsage": "listKeys(resourceId, apiVersion)",
```
[the remark this answers is not in the document]
Comment (bmoore-msft) [reply assets/ExpressionMetadata.json]: resourceId is correct, resourceName is shorthand.  The definitions were inconsistent and I may have over simplified - don't have a strong opinion (other than to be consistent).

[the remark this answers is not in the document]
Comment (StephenWeatherford) [reply assets/ExpressionMetadata.json]: The question is, can you pass in just a resource name and have it work?  If no, stick with resourceId, thanks.

[the remark this answers is not in the document]
Comment (StephenWeatherford) [reply assets/ExpressionMetadata.json]: I'm guessing the test failures are due to the resourceName/Id change.  The unit tests are over testing right now - I've loosened it up a bit but not enough.  Do you want me to handle fixing them?

[the remark this answers is not in the document]
Comment (bmoore-msft) [reply assets/ExpressionMetadata.json]: you can - provided that resource is defined in the template... if not, then you need the fulll resourceId.

**Comment by StephenWeatherford** [comment]
Thanks, Brian.

**Comment by bmoore-msft** [comment]
thx for fixing the tests ;)

**Pull request merged**